### PR TITLE
Properly log expiration date removal in audit log

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

File: apps/admin_audit/lib/Actions/Sharing.php [hunk 1]
```diff
@@ -320,15 +320,26 @@ public function updatePassword(array $params): void {
...
-			$params,
-			[
-				'itemType',
-				'itemSource',
-				'date',
-			]
-		);
+		if ($params['date'] === null) {
+			$this->log(
+				'The expiration date of the publicly shared %s with ID "%s" has been changed removed',
```
Comment (LukasReschke) [inline apps/admin_audit/lib/Actions/Sharing.php:325]: -changed

**Pull request merged**